packages/react-magma-dom/src/components/Table/Table.stories.tsx

```diff
@@ -868,4 +886,56 @@ export const NoRowsPerPageControl = args => {
       />
     </Card>
   );
-};
\ No newline at end of file
+};
+
+export const TableTitleWithNode = args => {
+  return (
+    <Table tableTitle={<Heading level={1}>Heading component example</Heading>} {...args}>
```

Comment:
Because `{...args}` is at the end, it's replacing tableTitle with the one that gets passed in so it reads `Row colors` instead of `Heading component example`